src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -783,7 +785,10 @@ private void delete() {
         getUndoManager().addEdit(compound);
 
         markBaseChanged();
-        frame.output(formatOutputMessage(Localization.lang("Deleted"), entries.size()));
+        frame.output(formatOutputMessage(!cut ? Localization.lang("Deleted") : Localization.lang("Cut"), entries.size()));
+
```

Comment:
Either you change the condition and put cut instead of not cut or you wrap it in if and else